src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -413,13 +411,6 @@ public override void DefaultVisit(Symbol symbol)
 
                 bool reportDiagnosticsForCurrentTrivia = trivia.SyntaxTree.ReportDocumentationCommentDiagnostics();
 
-                // If we're writing XML or we need to report diagnostics (either in this particular piece of trivia,
-                // or concerning undocumented [type] parameters), then we need to process this trivia node.
-                if (!(haveWriter || reportDiagnosticsForCurrentTrivia || reportParameterOrTypeParameterDiagnostics))
-                {
-                    continue;
-                }
```

Comment:
Do we not need an equivalent VB change?  DId it not have this problem?

Reply:
VB did not have this optimization, so part of the C# fix wasn't needed in VB.

---
In reply to: [184566119](https://github.com/dotnet/roslyn/pull/26425#discussion_r184566119) [](ancestors = 184566119)